ui/app/templates/components/secret-create-or-update.hbs

```diff
@@ -33,12 +33,6 @@
           @isMarginless={{true}}
         />
       {{/if}}
-      {{#if @modelForData.isFolder}}
```

Comment:
Removed this kv-specific logic, but this component along with `secret-edit` needs a cleanup round

Reply:
Should I make a ticket for this? Put it under one of the new tech-debt epics?

Reply:
Yeah I feel like we should revert this change - this was before we came to the consensus of removing all kv work in a separate PR